settings/checkstyle/checkstyle.xml

```diff
@@ -0,0 +1,350 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+  "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
+  "https://checkstyle.org/dtds/configuration_1_3.dtd">
+
+<module name="Checker">
```

Comment:
I have copied this file from [Armeria](https://github.com/line/armeria/) but removed checks related to Javadoc:

```
    <module name="JavadocType">
      <property name="scope" value="protected"/>
      <property name="excludeScope" value="package"/>
    </module>
    <module name="JavadocMethod">
      <property name="accessModifiers" value="protected, public"/>
      <property name="allowMissingParamTags" value="true"/>
      <property name="allowMissingReturnTag" value="true"/>
      <property name="allowedAnnotations" value="Override, Test"/>
    </module>
    <module name="MissingJavadocMethod">
      <property name="scope" value="protected"/>
    </module>
    <module name="MissingJavadocPackage"/>
    <module name="MissingJavadocType"/>
    <module name="JavadocParagraph"/>
    <module name="JavadocTagContinuationIndentation"/>
```